app/population/view/daily.njk

```diff
@@ -0,0 +1,114 @@
+{% extends "layouts/base.njk" %}
+{% from "panel/macro.njk" import appPanel %}
+{% from "data/macro.njk" import appData %}
+
+{%  macro statsEntry(property, value) %}
+  {%  set key = "population::daily_view.labels." + property %}
+  {%  set label = t(key) %}
+
+  <h4 class="govuk-heading-s">{{ label }}</h4>
+  <p class="govuk-body">{{ value }}</p>
+{% endmacro %}
+
+{% macro statsGroup(headingProperty, stats) %}
+  {%  set key = "population::daily_view.labels." + headingProperty %}
+  {%  set label = t(key) %}
+
+  <h3 class="govuk-heading-m">{{ label }}</h3>
+
+  {%  for stat in stats %}
+    {{  statsEntry(stat.property, stat.value) }}
+  {% endfor %}
+{% endmacro %}
+
+{% macro transfersGroup(headingProperty, transfers) %}
+  {%  set key = "population::daily_view.labels." + headingProperty %}
+  {%  set label = t(key) %}
+
+  <h3 class="govuk-heading-m">{{ label }}</h3>
+
+  <p class="govuk-body">
+    {{  transfers }}
```

Comment:
This feels a little similar to the macro above (`statsGroup`). Is there a way these could actually be shared or made to be similar?

Reply:
Macro has been removed and replaced with usage of `statsEntry`